### ISSUE TITLE
macOS scroll speed still not quite right

### ISSUE THREAD

**Issue by falkTX** [lv2/pugl#126]
After getting a release out of Cardinal with f7b3fe466e7c443b4ec60901899fc538b66c0cc1 in it, very soon after there came reports of scrolling now being too fast.
On this latest release I made it so the wheel/scroll is the same for all OSes.

See for the report
https://github.com/DISTRHO/Cardinal/issues/712

I can reproduce locally, happens with a intel macbook trackpad, I think this is the 2020 model.

What did you use for your own testing?

**Comment by falkTX** [comment]
I tested locally with a usb mouse and the same issue happens, though small wheel steps seem to behave correctly, only when scrolling a bit faster the "acceleration" makes the delta value go too high.

I can assume that is what you experienced and what your commit message refers to.

Issue is, and I can test with a trackpad, it is quite hard to make it move in very small steps and also very unintuitive.
We should see about a way to disable or reduce this acceleration, so it becomes consistent with other platforms.

**Comment by falkTX** [comment]
After some testing with cardinal with both trackpad and usb mouse, this is what feels comfortable and usable, compared to windows and linux:

```
  double dx = -[event scrollingDeltaX] / 2.0;
  double dy = [event scrollingDeltaY] / 2.0;
  if ([event hasPreciseScrollingDeltas]) {
    dx /= 10.0;
    dy /= 10.0;
  }
```

**Comment by quantumech3** [comment]
Yeah maybe the speed can just be changed to that if you are on MacOS. Another idea that might be more complicated to implement would be to put in a scroll bar allowing the user to change the sensitivity to their liking. But I don't know how hard it would be to change the UI.

**Comment by drobilla** [comment]
This part made the wheel of a physical mouse even further off from the "1.0 = one tick/line" of the other platforms:

      double dx = -[event scrollingDeltaX] / 2.0;
      double dy = [event scrollingDeltaY] / 2.0;

Given all the acceleration settings here, true consistency seems somewhere on the spectrum from very hard to literally impossible.

Your factors feel a lot more consistent with the touchpad for me though, so I applied that part in 8282e76, thanks.
